### ISSUE TITLE
Lifti: Deprecated filter `llms_get_quiz_theme_settings` logged when accessing the builder

### ISSUE THREAD

**Issue by eri-trabiccolo** [gocodebox/lifterlms-labs#16]
### Reproduction Steps
+ With the Divi theme enable "Lifti"
+ Access the course builder on a course with a quiz

### Expected Result
+ no deprecation warning logged in the llms-log

### Actual Result
+  deprecation warning logged in the llms-log

When the debug display is active this warning breaks the builder. In order to avoid this, customer can disable the debug display e.g. adding the following to their wp-config.php before` /* That's all, stop editing! Happy blogging. */`
```php
define( 'WP_DEBUG', true );​​​​​
define( 'WP_DEBUG_DISPLAY', false );​
```

### Error Log
`08-21-2020 @ 11:07:44 - Filter `llms_get_quiz_theme_settings` deprecated since 3.17.6. For more information see new methods at https://lifterlms.com/docs/course-builder-custom-fields-for-developers/
`

### System Info
```
LifterLMS 4.3.3
LifterLMS Labs 1.5.3
Divi *
```

**Comment by eri-trabiccolo** [comment]
Testing this with php 8.0 because of https://github.com/gocodebox/private-issues/issues/46 the way we call the `apply_filters_deprecated` throws a Fatal Error:
```
[28-Sep-2021 09:38:38 UTC] PHP Warning:  Undefined array key 0 in /shared/httpd/mywp/wordpress.git/wp-includes/plugin.php on line 233
[28-Sep-2021 09:38:38 UTC] PHP Fatal error:  Uncaught Error: Unknown named parameter $layout in /shared/httpd/mywp/wordpress.git/wp-includes/class-wp-hook.php:303
Stack trace:
#0 /shared/httpd/mywp/wordpress.git/wp-includes/plugin.php(233): WP_Hook->apply_filters(NULL, Array)
#1 /shared/httpd/mywp/wordpress.git/wp-includes/plugin.php(656): apply_filters_ref_array('llms_get_quiz_t...', Array)
#2 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/functions/llms.functions.quiz.php(140): apply_filters_deprecated('llms_get_quiz_t...', Array, '3.17.6')
#3 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/models/model.llms.quiz.php(253): llms_get_quiz_theme_setting('layout')
#4 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/abstracts/abstract.llms.post.model.php(1471): LLMS_Quiz->toArrayAfter(Array)
#5 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/models/model.llms.lesson.php(551): LLMS_Post_Model->toArray()
#6 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/abstracts/abstract.llms.post.model.php(1471): LLMS_Lesson->toArrayAfter(Array)
#7 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/models/model.llms.section.php(258): LLMS_Post_Model->toArray()
#8 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/abstracts/abstract.llms.post.model.php(1471): LLMS_Section->toArrayAfter(Array)
#9 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/models/model.llms.course.php(734): LLMS_Post_Model->toArray()
#10 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/abstracts/abstract.llms.post.model.php(1471): LLMS_Course->toArrayAfter(Array)
#11 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/admin/class.llms.admin.builder.php(619): LLMS_Post_Model->toArray()
#12 /Users/LjubaDavie/rock-dev/Dev/plugins/lifterlms/includes/admin/class.llms.admin.menus.php(264): LLMS_Admin_Builder::output()
#13 /shared/httpd/mywp/wordpress.git/wp-includes/class-wp-hook.php(303): LLMS_Admin_Menus->builder_init('')
#14 /shared/httpd/mywp/wordpress.git/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#15 /shared/httpd/mywp/wordpress.git/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#16 /shared/httpd/mywp/wordpress.git/wp-admin/admin.php(259): do_action('admin_page_llms...')
#17 {main}
```

**Comment by eri-trabiccolo** [comment]
fixed via #44, released with #39
